apps/api/test/seeders/day.seeder.ts

```diff
@@ -17,6 +17,11 @@ export class DaySeeder {
 
   static async createInDatabase(overrides: Partial<CreationAttributes<Day>> = {}): Promise<Day> {
     const seed = DaySeeder.create(overrides);
-    return await Day.create(seed);
+    try {
+      return await Day.create(seed);
+    } catch (error) {
+      console.error("Error creating Day in database:", error);
```

Comment:
question(non-blocking): why do you need this?

Reply:
Ouch, we don't. This was some debugging I added when my functional tests were failing and then totally forgot to remove. Thanks for the catch!